ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml

```diff
@@ -1269,6 +1269,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-disk-encryption
+- as: gcp-ipi-minimal-permission-p2-f28
+  cron: 23 12 4 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @network-openshiftsdn and not @fips'
```

Comment:
btw, the minimal permission is required to backport to 4.10?

Reply:
Yes, it works with 4.10. BTW, I referred to Yunfei's https://github.com/openshift/release/pull/38325/. Thanks!